frontend/src/metabase/setup/actions.ts

```diff
@@ -115,19 +120,16 @@ export const submitUser = createAsyncThunk<void, UserInfo, ThunkConfig>(
 
     MetabaseSettings.set("setup-token", null);
     dispatch(goToNextStep());
+    //  load the settings after the user is logged, needed later by setEmbeddingHomepageFlags
+    dispatch(reloadSettings());
```

Comment:
At this point the user is logged, and later on I'll need to access the full settings and "admin settings".
The admin settings are needed because I need to check if 'enable-embedding' is set via env variable, and in that case not to touch the setting/set the flag

Reply:
I agree, done in 69403b6973a47f0ef5f3e95e80e984516819cb02